drivers/npmx/npmx_shell.c

```diff
@@ -4325,76 +4125,70 @@ SHELL_STATIC_SUBCMD_SET_CREATE(sub_gpio, SHELL_CMD(config, &sub_gpio_config, "GP
+SHELL_STATIC_SUBCMD_SET_CREATE(sub_ldsw_active_discharge,
+			       SHELL_CMD(set, NULL, "Set active discharge status",
+					 cmd_ldsw_active_discharge_set),
+			       SHELL_CMD(get, NULL, "Get active discharge status",
+					 cmd_ldsw_active_discharge_get),
 			       SHELL_SUBCMD_SET_END);
 
-/* Creating dictionary subcommands (level 3 command) array for command "sub_ldsw_ldo_voltage". */
+/* Creating dictionary subcommands (level 3 command) array for command "ldsw gpio". */
+SHELL_STATIC_SUBCMD_SET_CREATE(sub_ldsw_gpio,
+			       SHELL_CMD(set, NULL, "Set LDSW GPIO", cmd_ldsw_gpio_set),
+			       SHELL_CMD(get, NULL, "Get LDSW GPIO", cmd_ldsw_gpio_get),
+			       SHELL_SUBCMD_SET_END);
+
+/* Creating dictionary subcommands (level 3 command) array for command "ldsw ldo_voltage". */
 SHELL_STATIC_SUBCMD_SET_CREATE(sub_ldsw_ldo_voltage,
 			       SHELL_CMD(set, NULL, "Set LDO voltage", cmd_ldsw_ldo_voltage_set),
 			       SHELL_CMD(get, NULL, "Get LDO voltage", cmd_ldsw_ldo_voltage_get),
 			       SHELL_SUBCMD_SET_END);
 
-/* Creating subcommands (level 4 command) array for command "ldsw soft_start enable". */
-SHELL_STATIC_SUBCMD_SET_CREATE(sub_ldsw_soft_start_enable,
-			       SHELL_CMD(get, NULL, "Get soft-start enable status",
-					 cmd_ldsw_soft_start_enable_get),
-			       SHELL_CMD(set, NULL, "Set soft-start enable status",
-					 cmd_ldsw_soft_start_enable_set),
+/* Creating dictionary subcommands (level 3 command) array for command "ldsw mode". */
+SHELL_STATIC_SUBCMD_SET_CREATE(sub_ldsw_mode,
+			       SHELL_CMD(set, NULL, "Set LDSW mode", cmd_ldsw_mode_set),
+			       SHELL_CMD(get, NULL, "Get LDSW mode", cmd_ldsw_mode_get),
+			       SHELL_SUBCMD_SET_END);
+
+/* Creating subcommands (level 4 command) array for command "ldsw soft_start status". */
+SHELL_STATIC_SUBCMD_SET_CREATE(sub_ldsw_soft_start_status,
+			       SHELL_CMD(set, NULL, "Set soft-start status",
+					 cmd_ldsw_soft_start_status_set),
+			       SHELL_CMD(get, NULL, "Get soft-start status",
+					 cmd_ldsw_soft_start_status_get),
 			       SHELL_SUBCMD_SET_END);
 
 /* Creating subcommands (level 4 command) array for command "ldsw soft_start current". */
 SHELL_STATIC_SUBCMD_SET_CREATE(sub_ldsw_soft_start_current,
-			       SHELL_CMD(get, NULL, "Get soft-start current status",
-					 cmd_ldsw_soft_start_current_get),
-			       SHELL_CMD(set, NULL, "Set soft-start current status",
+			       SHELL_CMD(set, NULL, "Set soft-start current",
 					 cmd_ldsw_soft_start_current_set),
+			       SHELL_CMD(get, NULL, "Get soft-start current",
+					 cmd_ldsw_soft_start_current_get),
 			       SHELL_SUBCMD_SET_END);
 
 /* Creating dictionary subcommands (level 3 command) array for command "ldsw soft_start". */
 SHELL_STATIC_SUBCMD_SET_CREATE(sub_ldsw_soft_start,
-			       SHELL_CMD(enable, &sub_ldsw_soft_start_enable, "Soft-start enable",
+			       SHELL_CMD(status, &sub_ldsw_soft_start_status, "Soft-start status",
```

Comment:
Why change it to `status`?
If anything, this should be moved under `sub_ldsw_status`.

Reply:
https://github.com/NordicSemiconductor/npmx-zephyr/pull/73#discussion_r1366785865